### PR TITLE
[cli] Update @vercel/fun@1.1.0

### PR DESCRIPTION
**Pull request by cb1kenobi**
This PR updates `@vercel/fun` to v1.1.0 which includes a fix to `setPrototypeOf()` in the error handling code which `ncc` was clobbering.

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -8085,6 +8085,7 @@ packages:
 
   /debug@4.1.1:
     resolution: {integrity: sha512-pYAIzeRo8J6KPEaJ0VWOh5Pzkbw/RetuzehGM7QRRX5he4fPHx2rdKMB256ehJCkX+XRQm16eZLqLNS8RSZXZw==}
+    deprecated: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
```
Comment (cb1kenobi) [inline pnpm-lock.yaml:8088]: @styfle why did `pnpm` add these deprecation messages to the lock file?

Comment (styfle) [reply pnpm-lock.yaml:8088]: Not sure. Could have changed since we swapped pnpm versions

Comment (styfle) [reply pnpm-lock.yaml:8088]: https://github.com/pnpm/pnpm/issues/5772#issuecomment-1344807194

File: .changeset/large-pets-vanish.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'vercel': minor
```
Comment (styfle) [inline .changeset/large-pets-vanish.md]: Why is this minor? Is this adding a new feature?

**Review by TooTallNate — DISMISSED** [review]

File: .changeset/large-pets-vanish.md (hunk 2 above)
Comment (cb1kenobi) [reply .changeset/large-pets-vanish.md]: Habit

Comment (cb1kenobi) [reply .changeset/large-pets-vanish.md]: Fixed

**Review by styfle — APPROVED** [review]

**Pull request merged**